docs/learn/smart-contract-developers/create-lsp7-token.md

```diff
@@ -93,40 +93,32 @@ const config: HardhatUserConfig = {
 export default config;
 ```
 
-### Create a deploy script
+### Create a deployment script
 
-We will create a script to deploy the smart contract to the LUKSO Testnet network. You can either use a regular EOA (Externally Owned Account) or a Universal Profile. Let's see those 2 possibilities below.
+Create a script to deploy the smart contract to the LUKSO Testnet network. You can either use a regular EOA (Externally Owned Account) or a Universal Profile.
```

Comment:
So this is a script that it is intended to be run not only via the CLI, but in a frontend as well, which will trigger the Browser Extension?

Reply:
No, this is mainly intended to create a deployment script.